src/lib/edge-functions/proxy.js

```diff
@@ -141,34 +148,49 @@ const prepareServer = async ({
   port,
   projectDir,
 }) => {
-  const bundler = await import('@netlify/edge-bundler')
-  const distImportMapPath = getPathInProject([DIST_IMPORT_MAP_PATH])
-  const runIsolate = await bundler.serve({
-    ...getDownloadUpdateFunctions(),
-    certificatePath,
-    debug: env.NETLIFY_DENO_DEBUG === 'true',
-    distImportMapPath,
-    formatExportTypeError: (name) =>
-      `${NETLIFYDEVERR} ${chalk.red('Failed')} to load Edge Function ${chalk.yellow(
-        name,
-      )}. The file does not seem to have a function as the default export.`,
-    formatImportError: (name) => `${NETLIFYDEVERR} ${chalk.red('Failed')} to run Edge Function ${chalk.yellow(name)}:`,
-    importMaps,
-    inspectSettings,
-    port,
-  })
-  const registry = new EdgeFunctionsRegistry({
-    bundler,
-    config,
-    configPath,
-    directories,
-    getUpdatedConfig,
-    internalFunctions,
-    projectDir,
-    runIsolate,
-  })
+  try {
+    const bundler = await import('@netlify/edge-bundler')
+    const distImportMapPath = getPathInProject([DIST_IMPORT_MAP_PATH])
+    const runIsolate = await bundler.serve({
+      ...getDownloadUpdateFunctions(),
+      certificatePath,
+      debug: env.NETLIFY_DENO_DEBUG === 'true',
+      distImportMapPath,
+      formatExportTypeError: (name) =>
+        `${NETLIFYDEVERR} ${chalk.red('Failed')} to load Edge Function ${chalk.yellow(
+          name,
+        )}. The file does not seem to have a function as the default export.`,
+      formatImportError: (name) =>
+        `${NETLIFYDEVERR} ${chalk.red('Failed')} to run Edge Function ${chalk.yellow(name)}:`,
+      importMaps,
+      inspectSettings,
+      port,
+    })
+    const registry = new EdgeFunctionsRegistry({
+      bundler,
+      config,
+      configPath,
+      directories,
+      getUpdatedConfig,
+      internalFunctions,
+      projectDir,
+      runIsolate,
+    })
 
-  return { registry, runIsolate }
+    return { registry, runIsolate }
+  } catch (error) {
+    log(
+      `${chalk.red(
+        BANG,
+      )} There was a problem setting up the Edge Functions environment and it's unfortunately not possible to run Edge Functions from the CLI on this platform. More on supported platforms here: https://deno.land/manual/getting_started/installation. Error: ${
```

Comment:
We try to generate a `ntl.fyi` shortlink for user-facing links.

Reply:
is `ntl.fyi/install-deno` good enough?

Reply:
Yeah! 🚀

Reply:
asked the marketing team [here](https://netlify.slack.com/archives/C1EENQ18U/p1655214472502759)